CCLocation/ColocatorManager.swift

```diff
@@ -139,9 +218,15 @@ class ColocatorManager {
     }
     
     deinit {
```

Comment:
Might be worth to check whether we get into the deinit and if the database variables are deallocated. If not, that could explain the extra references on stop.

Reply:
I found what the issue is. ColocatorManager is a shared instance. It has references to 3 databases and this is the cause of what we saw on Memory Inspector.

Reply:
Great, that is what I thought.